Makefile

```diff
@@ -25,7 +25,7 @@ iteration=0
 local_code_path=0
 openresty="openresty"
 artifact="0"
-apisix_repo="https://github.com/apache/apisix"
+apisix_repo="https://github.com/Revolyssup/apisix"
```

Comment:
Why do you still use your own repo in the official PR?

Reply:
After step 2 here[1] is complete, I will change the link here as per step 3.

1. https://github.com/api7/apisix-build-tools/pull/345#discussion_r1378495957